resources/ory/charts/hydra/charts/hydra-maester/values.yaml

```diff
@@ -1,24 +1,28 @@
+# -- Number of replicas in deployment
```

Comment:
Hydra-maester update changes will be added in separate PR: https://github.com/kyma-project/kyma/pull/14301

Reply:
Please remove charts/hydra-maester from this PR